eng/common/scripts/job-matrix/job-matrix-functions.ps1

```diff
@@ -202,6 +202,7 @@ function GetMatrixConfigFromFile([String] $config)
 
 function GetMatrixConfigFromYaml([String] $yamlConfig)
 {
+    Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module
```

Comment:
You will have to import the common modules since `Install-ModuleIfNotInstalled` is defined by us.

https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1#L6

```
. (Join-Path $PSScriptRoot "../Helpers" PSModule-Helpers.ps1)
```